README.md

```diff
@@ -16,13 +16,13 @@ Trolleybus Simulator addon for Garry's Mod.
 
 Main features:
 * Well-developed trolleybuses
-* Contact network system
+* Catenary network system
 * Routes system: trolleybus stops and passengers
 * Simple traffic system
 * Map created for trolleybuses
 * Editor tools for creating infrastructure for trolleybuses
 
-Mechanics of trolleybuses based on real world. Controls contains many subsystems (engine, circuits, informers, control systems, etc). This will make you feel like a real trolleybus driver.
+Mechanics of trolleybuses based on real world. The interactive elements consist of many subsystems (engine, circuits, informers, control systems, etc). This will make you feel like a real trolleybus driver.
```

Comment:
`Controls` means here not just interactive elements, but also technical elements (like engine, circuits).

Reply:
Come to think of it, I'd change it to "The simulated elements" instead.